include/databaseFunctions.php

```diff
@@ -1,34 +1,24 @@
 <?php
 	include('config/config.php');
 	include('include/db_query.php');
-	//POST FUNCTIONS
-	function InsertBlogPost($author, $date, $title, $body){
-		$result = dbQuery("
-			INSERT INTO posts (author, date, title, body)
-			VALUES('$author', '$date', '$title', '$body')
-		")->fetch();
-	}
 
-	function GetBlogPost($postID){
-		$result = dbQuery("
-			SELECT *
-			FROM posts
-			WHERE postID = $postID
-		")->fetch();
-		return $result;
+	function home(){
+		echo"<a href='index.php'>back to home</a>";
 	}
-
-	function DeleteBlogPost($postID){
+	
+	//POST FUNCTIONS
+	function InsertBlogPost($author, $title, $body){
 		$result = dbQuery("
-			DELETE FROM posts
-			WHERE postID = $postID
+			INSERT INTO posts (author, title, body, isPic)
```

Comment:
Having an `isPic` field isn't a problem necessarily, but it's not super flexible. If you ever want to support more than just posts and pictures, this will require major changes.

An alternative would be to have a "type" field where you can store the string "post" or "pic". Even more robust (although maybe overengineered) would be to have another table called "content_types" or something like that, and have the posts table link to it with a `ContentTypesId` field. This would create a one-to-many relationship.